Kernel/Graphics/Intel/NativeDisplayConnector.cpp

```diff
@@ -41,25 +41,28 @@ ErrorOr<void> IntelNativeDisplayConnector::create_attached_framebuffer_console(B
     return {};
 }
 
-IntelNativeDisplayConnector::IntelNativeDisplayConnector(IntelDisplayConnectorGroup const& parent_connector_group, PhysicalAddress framebuffer_address, size_t framebuffer_resource_size)
+IntelNativeDisplayConnector::IntelNativeDisplayConnector(IntelDisplayConnectorGroup const& parent_connector_group, Type type, PhysicalAddress framebuffer_address, size_t framebuffer_resource_size)
     : DisplayConnector(framebuffer_address, framebuffer_resource_size, true)
+    , m_type(type)
     , m_parent_connector_group(parent_connector_group)
 {
 }
 
-ErrorOr<void> IntelNativeDisplayConnector::set_y_offset(size_t)
+void IntelNativeDisplayConnector::set_edid_bytes(Badge<IntelDisplayConnectorGroup>, Array<u8, 128> const& raw_bytes)
 {
-    return Error::from_errno(ENOTIMPL);
+    // Note: The provided EDID might be invalid (because there's no attached monitor)
+    // Therefore, set might_be_invalid to true to indicate that.
+    DisplayConnector::set_edid_bytes(raw_bytes, true);
 }
 
-ErrorOr<void> IntelNativeDisplayConnector::unblank()
+ErrorOr<void> IntelNativeDisplayConnector::set_y_offset(size_t)
 {
     return Error::from_errno(ENOTIMPL);
 }
 
-void IntelNativeDisplayConnector::set_edid_bytes(Badge<IntelDisplayConnectorGroup>, Array<u8, 128> const& edid_bytes)
+ErrorOr<void> IntelNativeDisplayConnector::unblank()
```

Comment:
`Kernel/Graphics: Prepare Intel driver to support more generations` shuffles set_edid_bytes, set_y_offset and unblank around in this file, which makes the actual diff hard to follow.